### PR TITLE
Add support for ChromeOS

### PR DESCRIPTION
**Pull request by nathanchance**
Each change should be self explanatory, let me know if there are any problems or improvements!

### CODE REVIEW

File: generator.yml [hunk 1]
```diff
@@ -178,6 +189,9 @@ targets:
   - &default     {targets: [default]}
   - &kernel      {targets: [kernel]}
   - &kernel_dtbs {targets: [kernel,dtbs]}
+chromeos_configs:
+  - &arm64-cros-configs  {config: [chromeos/config/chromeos/base.config, chromeos/config/chromeos/arm64/common.config, chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config]}
+  - &x86_64-cros-configs {config: [chromeos/config/chromeos/base.config, chromeos/config/chromeos/x86_64/common.config, chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config]}
```
Comment (nickdesaulniers) [inline generator.yml:194]: Is this something tuxbuild supports that we simply haven't used before (using multiple config files?)

File: utils.py [hunk 2]
```diff
@@ -73,6 +73,11 @@ def get_cbl_name():
         if "archlinux" in base_config:
             return "x86_64"
 
+    # ChromeOS configurations have the architecture as the second to last
+    # folder of the second config fragment path
```
Comment (nickdesaulniers) [inline utils.py:77]: Can you put an example in the comment? That makes it easier to review what this is doing.

**Review by nickdesaulniers — APPROVED** [review]
Very nice!

Let's land this, but let's discuss more with @groeck what other branches we should be testing.  For example, I suspect that CrOS still supports branches of stable older than 5.10.

Thanks @nathanchance !! 💯

File: generator.yml (hunk 1 above)
Comment (nathanchance) [reply generator.yml:194]: Yes, although using a config fragment as the first ("base") configuration is new: https://gitlab.com/Linaro/tuxmake/-/merge_requests/269

File: utils.py (hunk 2 above)
Comment (nathanchance) [reply utils.py]: Done.

**Comment by nathanchance** [comment]
> For example, I suspect that CrOS still supports branches of stable older than 5.10.

They do but the configuration structure is different from 5.4 and earlier to 5.10 and newer and I am not really sure how to handle that within `generator.yml`.

```
$ git switch chromeos-5.4
...

$ fd . chromeos/config | sort
chromeos/config/arm64
chromeos/config/arm64/chromiumos-arm64.flavour.config
chromeos/config/arm64/chromiumos-mediatek.flavour.config
chromeos/config/arm64/chromiumos-qualcomm.flavour.config
chromeos/config/arm64/chromiumos-rockchip64.flavour.config
chromeos/config/arm64/common.config
chromeos/config/armel
chromeos/config/armel/chromiumos-arm.flavour.config
chromeos/config/armel/chromiumos-rockchip.flavour.config
chromeos/config/armel/common.config
chromeos/config/base.config
chromeos/config/x86_64
chromeos/config/x86_64/chromeos-amd-stoneyridge.flavour.config
chromeos/config/x86_64/chromeos-intel-denverton.flavour.config
chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
chromeos/config/x86_64/chromeos-x86_64-reven.flavour.config
chromeos/config/x86_64/chromiumos-x86_64.flavour.config
chromeos/config/x86_64/common.config

$ git switch chromeos-5.10
...

$ fd . chromeos/config | sort
chromeos/config/chromeos
chromeos/config/chromeos/arm64
chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config
chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
chromeos/config/chromeos/arm64/chromiumos-qualcomm.flavour.config
chromeos/config/chromeos/arm64/chromiumos-rockchip64.flavour.config
chromeos/config/chromeos/arm64/common.config
chromeos/config/chromeos/armel
chromeos/config/chromeos/armel/chromiumos-arm.flavour.config
chromeos/config/chromeos/armel/chromiumos-rockchip.flavour.config
chromeos/config/chromeos/armel/common.config
chromeos/config/chromeos/base.config
chromeos/config/chromeos/x86_64
chromeos/config/chromeos/x86_64/chromeos-amd-stoneyridge.flavour.config
chromeos/config/chromeos/x86_64/chromeos-intel-denverton.flavour.config
chromeos/config/chromeos/x86_64/chromeos-intel-pineview.flavour.config
chromeos/config/chromeos/x86_64/chromeos-x86_64-reven.flavour.config
chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config
chromeos/config/chromeos/x86_64/common.config
chromeos/config/manatee
chromeos/config/manatee/base.config
chromeos/config/manatee/x86_64
chromeos/config/manatee/x86_64/chromeos-intel-pineview.flavour.config
chromeos/config/manatee/x86_64/common.config
chromeos/config/termina
chromeos/config/termina/arm64
chromeos/config/termina/arm64/common.config
chromeos/config/termina/arm64/container-vm-arm64.flavour.config
chromeos/config/termina/base.config
chromeos/config/termina/x86_64
chromeos/config/termina/x86_64/common.config
chromeos/config/termina/x86_64/container-vm-x86_64.flavour.config
```

**Pull request merged**